test/Differentiation/differentiable_protocol.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-typecheck-verify-swift
+
```

Comment:
Hmm. I think you'll need a `lit` configuration option `autodiff` and ` // REQUIRES: autodiff` on this test?

Reply:
Thanks for the catch! Added a `lit` option called `differentiable_programming` in https://github.com/apple/swift/pull/27511/commits/5aececee414f9a56adfed3f751a24df2d4a08c5e.